README.md

```diff
@@ -3,6 +3,11 @@
 A curated list of papers and open source resources focused on 3D Gaussian Splatting, intended to keep pace with the anticipated surge of research in the coming months. If you have any additions or suggestions, feel free to contribute. Additional resources like blog posts, videos, etc. are also welcome.
 
 ### Update Log:
+- **October 18, 2023**: 
+  - Added Github page link for Real-time Photorealistic Dynamic Scene Representation.
+  - Added other unofficial implementations based on the original paper.
+  - Added Nerfstudio, Blender, WebRTC, iOS & Metal viewers.
+  - Added Paid Implementations - Unreal Plugin and Tellusim.
```

Comment:
I think it could be beneficial to have some awareness and list but make clear of Paid / Closed Source resources, the gsplat.tech viewer could also fall into this category https://github.com/MrNeRF/awesome-3D-gaussian-splatting/pull/4#pullrequestreview-1686052466

There are also other closed / wip VR viewers that could be mentioned there, and then moved if there is open source code released

Reply:
I agree that if they become open source, they should be listed. However, if we begin listing every viewer or tech demo, our list will quickly become arbitrary. Why not include the Luma or Polycam viewer? How do they differ from gsplat.tech? Is there any confirmation that the gsplat.tech viewer wasn't acquired by Luma?

Everything is closed source, so we can't derive any educational value from it. These tools are presented on great platforms that I enjoy, much like Twitter, YouTube, etc. That's fine for passive consumption. 

I've also considered removing the 4DGS paper, as I couldn't reproduce any of the showcased results: https://github.com/hustvl/4DGaussians/issues/7. Hence, I remain skeptical.

Reply:
Ok yes I see the point. I did actually update the commit and added Luma, Polycam etc but the only thing I think to keep on the list is the Unreal plugin via Volinga method that I will put under viewers, everything else I'll remove.

The new Nerfstudio gsplat rasterizer, should that actually go under unofficial implementations section? I presume your fast cuda code is under the Training section meaning that section is anything built on top of / modified from the original code?

Reply:
So, Luma and Polycam have been removed, haven't they? Please avoid commercial applications, as they can be googled in just a few seconds, and I don't want to advertise anything. Additionally, I'd prefer to exclude anything without source code (research papers can be an exception).

I'll draft some contribution guidelines in the next few days. I didn't expect this repo to gain popularity so quickly.

What about volinga.ai? Is the plugin free? Does it offer source code?

Yeah, I believe the NerfStudio gsplat implementation should be in the unofficial section. This one is intriguing; I doubt it's a clean rewrite. Just examine the structure of the rasterizer and the git logs. They didn't include a license — or rather, they removed the Apache license...

My repo can also be listed there. It's essentially a reimplementation of the original source code but much faster. So, yes, it's unofficial.

Reply:
"What about volinga.ai? Is the plugin free? Does it offer source code?"
It is free but limited and non-commercial but no source code.

If you want this repo to be strictly open source can filter out the links that aren't no problem.

I'll tidy up the commit with these suggests and do a pr.